api.py

```diff
@@ -0,0 +1,116 @@
+import os
+import threading
+import enum
+from time import sleep
+from subprocess import Popen, DEVNULL
+
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+
+from pytube import YouTube
+from pytube.exceptions import AgeRestrictedError, VideoUnavailable, RegexMatchError
```

Comment:
same thing, just use `pytube.exceptions.AgeRestrictedError` instead of importing `AgeRestrictedError` directly, makes reading code easier